lessons/en/misc/plug.md

```diff
@@ -311,12 +311,23 @@ It's considered good practice to make the port configurable by putting it in a c
 
 We'll set an application environment variable in `config/config.exs`
 
+For Elixir versions below 1.9:
+
 ```elixir
 use Mix.Config
 
 config :example, cowboy_port: 8080
 ```
 
+For Elixir version 1.9+ (`Mix.Config` was removed in Elixir 1.9):
```

Comment:
```suggestion
For Elixir version 1.9+ (`Mix.Config` was deprecated in Elixir 1.9):
```

Correct me if I am wrong, but I believe this was not removed just deprecated with 1.9